front/lib/api/assistant/conversation.ts

```diff
@@ -45,6 +51,68 @@ export async function* postUserMessage(
   | AgentMessageTokensEvent
   | AgentMessageSuccessEvent
 > {
+  const user = auth.user();
+  if (!user) {
+    // TBD
```

Comment:
@spolu do we want to throw here ? Or yield some kind of error event and return ?

Reply:
No you can definitely post a user message by API with no `user`

Reply:
ok thanks